website/integrations/services/apple/index.md

```diff
@@ -0,0 +1,295 @@
+> Unlike Apple Business Manager, it includes built-in mobile device management capabilities without requiring third-party MDM solutions, making it ideal for organizations with limited IT resources.
+>
+> -- [Apple Business Essentials](https://www.apple.com/business/essentials/)
+
+:::tip
+
+While this guide shows how to configure authentik as the identity provider for Apple Business Manager,
+the same steps can be used to configure both Apple School Manager and Apple Business Essentials.
+
+:::
+
+## Authentication Flow
+
+This sequence diagram shows a high-level flow between the user's MacBook, authentik, and Apple.
+
+```mermaid
+sequenceDiagram
+    autonumber
+    participant User
+    participant authentik
+    participant Apple
+
+    User->>Apple: User sign-in via iCloud
+    Note over Apple: 🔍 Email domain is federated
+    Apple-->>authentik: Redirect to authentik
+    Note over authentik: ✅ Authenticate
+    authentik-->>Apple: Identity verified
+    Apple-->>User: Device enrolled!
+
+```
+
+In short, Apple Business Manager recognizes the email domain
+as a federated identity provider, eliminating the need for separate Apple ID credentials.
+
+---
+
+## Preparation
+
+By the end of this integration, your users will be able to enroll their Apple devices using their authentik credentials.
+
+You'll need to have authentik instance running and accessible on an HTTPS domain, and an Apple Business Manager user with the role of Administrator or People Manager.
+
+:::warning{title="Caveats"}
+
+Be aware that Apple Business Manager imposes the following restrictions on federated authentication:
+
+- Federated authentication should use the user’s email address as their user name. Aliases aren’t supported.
+- Existing users with an email address in the federated domain will automatically be converted to federated authentication, effectively _taking ownership_ of the account.
+- User accounts with the role of Administrator, Site Manager, or People Manager can’t sign in using federated authentication; they can only manage the federation process.
+
+:::
+
+### Placeholders
+
+The following placeholders are used:
+
+- `authentik.company`: The Fully Qualified Domain Name of the authentik installation.
+
+## authentik Configuration
+
+Before you can proceed with the Apple Business Manager configuration, you'll need to configure authentik to act as an identity provider for Apple. Start by logging into your authentik instance and navigating to the Admin interface.
+
+### Scope Mapping
+
+In the Admin interface, navigate to **Customization** -> **Property Mappings**.
+
+Press **Create** and when prompted for a choice of mapping, select **Scope Mapping**.
+
+#### Apple Business Manager Profile
+
+Create a new scope mapping to handle the user's profile information:
+
+| Field      | Value                          |
```

Comment:
Again for consistency reasons, I'd rather not use a table for only two values. Yet another topic we can discuss separately, for longer lists of values.